src/parquet/util/test-common.h

```diff
@@ -183,6 +183,15 @@ void random_byte_array(int n, uint32_t seed, uint8_t *buf,
     buf += out[i].len;
   }
 }
+
+void random_levels(int n, uint32_t seed, int16_t max_level, int16_t* out) {
+  std::mt19937 gen(seed);
+  std::uniform_int_distribution<uint16_t> d(0, max_level);
+  for (int i = 0; i < n; ++i) {
+    out[i] = d(gen);
+  }
+}
```

Comment:
changing it to `random_int` breaks the templates in the tests. Ideally we have to combine `real` and `int` generators into a single `random_numbers` template where `std::uniform_int_distribution` is called for `int16_t` `int32_t` `int64_t`and `std::uniform_real_distribution` is called for `float` and `double`.